### DISCUSSION TITLE
Microphone, RPI 4B and power source

### DISCUSSION THREAD

**Discussion by Fish31520** [mcguirepr89/BirdNET-Pi#567] [Q&A] [answered]
Hi,

I'm really new to RPI and birds recording/identification.

So far everything is fine except the fact that I'm using a USB powered microphone and sometime it is doing some fishy "electric" noises.

I would like to change it but most of the mics I've seen so far need an external power supply (AA battery or Phantom power supply).

So I'm wondering if there are mics (100€/$ top) which can be powered by the 3.5 jack output of the RPI 4B.

Thanks !

Jean

**Comment by nilspupils** [comment, marked as answer]
Hi @Fish31520, 

without going into too many details, let me recommend the Boya M3. It is powered by USB but doesn't have the noise problems, since it works digitally. It also comes with a 6m cable.

https://www.amazon.de/BOYA-Digitales-USB-Typ-C-Lavalier-Mikrofon-Android-Ger%C3%A4te-Einzelkopf/dp/B08H1WXH5B
(You will need THIS SPECIFIC MODEL)!

 You will need an additional adapter from USB-A to USB-C. 
https://www.amazon.de/Syntech-Adapter-Compatible-Standard-Interface/dp/B085VT1VJT


There is also a very nice (but very long) discussion about microphones here: 
https://github.com/mcguirepr89/BirdNET-Pi/discussions/39

Good luck and please report back!
Nils

**Reply by Fish31520** [reply]
Thanks !

I didn't know such a small microphone (lavalier microphone) would do the job and be able to record birds songs.

But could it be the USB plug from the RPI which create all those funny noises that I've got, or could it come from the mic itself (https://www.t-nb.com/en/product-catalog/multimedia/influence/usb-influence-accessories/usb-omnidirectional-microphone-influence) ?

**Reply by nilspupils** [reply]
I have struggled with noisy microphones for some months, and although you are right the USB bus can be noisy it is not the source of the microphone noise. The main reason for it being common mode noise from the analog lines. With the Boya you will have a great microphone with little to no noise at all, since it communicates with a digital standard. It has a very good sensitivity and good self-noise but will not be right on par with the AOM-5024, which is best of its class.

**Reply by Fish31520** [reply]
But if I get it, in both cases (your setup and the one described there : https://github.com/mcguirepr89/BirdNET-Pi/discussions/39), you'll use the USB plugs instead of the 3.5 jack plug ?

**Reply by Fish31520** [reply]
Thanks a lot for your patience and your help, I'll go with the mic you're recommending me (now I just need to be patient ! 😅).

Thanks !

**Reply by nilspupils** [reply]
It is not even possible to connect a microphone to the pi at the 3,5mm jack, it just doesn't have audio input functionality and also doesn't provide a bias voltage (plug-in-power) both of which would be needed. It can be used for audio output (in a really bad quality, since it doesn't even use a proper DAC).

**Reply by Fish31520** [reply]
Ok, so we've got no other choice but to use the USB plug (+ USB sound card).
Thanks ! 😉

**Comment by w0mbatian** [comment]
I had a Zoom H1 recorder on hand and it is working very well. Powered by USB (its own battery is not necessary).
It has a USB microphone mode.
It has a reputation for very high quality recordings  About $100 US or under $200 Australian.
I have it well separated from the Pi 4 with a 5 metre active USB extension cable. I can hear no unwanted noises.
The current model is called Zoom H1N
Ian

**Reply by Fish31520** [reply]
Thanks a lot !

I gonna take a look at it !

**Comment by Fish31520** [comment]
Hi Guys,

I tried to follow all the advices found on this hub and I've got to admit that my recordings are much better now that I use a Lavalier mic and a USB sound card.

For those who are reading me from Europe, I bought this mic https://www.thomann.de/fr/rode_lavalier_go.htm and this USB sound card https://www.ldlc.com/fiche/PB00192204.html?offerId=AR201507290039

I'm far from being an expert but clearly the recordings I've got now are really clearer.

Thanks once again to those who helped me !
